lang/cs/Org.Apache.REEF.IO/FileSystem/AzureDatLake/AzureDataLakeFileSystem.cs

```diff
@@ -1,83 +0,0 @@
-﻿using System;
```

Comment:
This could be github's code review viewing, but make sure to use "git mv" instead of deleting and reading a file. This will preserve proper change history. Since this branch will get squashed it doesn't matter but remember for the future.

Reply:
sure